core/src/trezor/lvglui/i18n/locales/ja.py

```
@@ -820,7 +820,7 @@
     "アカウントを選択",
     "アカウントに移動",
     "入力フォーマットエラー",
-    "ERC-20 トークンは Ethereum ネットワーク上で発行されます。つまり、ETH ウォレット アドレスを使用して、任意の ERC-20 トークンを受け取ることができます。",
+    "あなたのアドレスはEVMネットワークアドレスです。他のEVM互換ネットワーク（Ethereum、BNB Chain、Polygon、Arbitrum One、Avalancheなど）で資産を管理するために使用できます。",
```

Ruff (0.8.2)

String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)